CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+## [1.1.4](https://github.com/ably/ably-php/tree/1.1.5) (2022-02-23)
+[Full Changelog](https://github.com/ably/ably-php/compare/1.1.4...1.1.5)
+
+**Fixed bugs:**
+
+- Using a clientId should no longer be forcing token auth in the 1.1 spec [\#74](https://github.com/ably/ably-php/issues/74)
+
+**Closed issues:**
+
+- ably php issue [\#131](https://github.com/ably/ably-php/issues/131)
+- Installing removes Symfony-Packages [\#106](https://github.com/ably/ably-php/issues/106)
+- Create code snippets for homepage \(php\) [\#105](https://github.com/ably/ably-php/issues/105)
+- The library should not be throwing an exception when a message is sent with different `clientId` from `auth.clientId` [\#101](https://github.com/ably/ably-php/issues/101)
+- TokenRequest: timestamp is integer in PHPDoc, string at runtime, Flutter expects int [\#99](https://github.com/ably/ably-php/issues/99)
+- Batch Mode, sending multiple messages to multiple channels [\#98](https://github.com/ably/ably-php/issues/98)
+
+**Merged pull requests:**
+
+- Fix setting X-Ably-\* and Accept headers [\#133](https://github.com/ably/ably-php/pull/133) ([lmars](https://github.com/lmars))
+- Add standard "About Ably" info to all public repos [\#130](https://github.com/ably/ably-php/pull/130) ([marklewin](https://github.com/marklewin))
```

Comment:
This is a good example of the kind of line item which we should manually remove from these change logs at the release PR stage, as it adds zero value to customers.

Reply:
Good point, I've removed it now